src/client/storage/index-arlocal.js

```diff
@@ -64,12 +63,10 @@ const filesDir = path.join(resolveHome(config.get('datadir')), config.get('stora
 
 let arweave;
 // load the arweave key for arlocal
-const keystorePath = path.join(resolveHome(config.get('datadir')), config.get('wallet.keystore_path'));
+const keystorePath = config.get('wallet.keystore_path');
```

Comment:
This should also needs `resolveHome`, since the path may be inside the home dir `~`. Home dir doesn't work when running the executable, that's why we need `resolveHome` call.